test/evergreen.yml

```diff
@@ -5474,7 +5474,7 @@ buildvariants:
 - name: little-endian
   display_name: "~ Little-endian (x86)"
   run_on:
-  - ubuntu1804-large
+  - ubuntu2204-large
```

Comment:
I'm 90% sure this platform is little endian, but could we get confirmation?

Additionally is there a reason we're gone with 2204 here and while the other platforms are 2004? It's not an issue I'm just curious as to the reason

Reply:
Same question for 2204 vs 2004.

Reply:
I triggered a patch build with the ubuntu2004-large distro and it works fine. I used 2204 just because it the latest distro version available.

Reply:
Got it, thanks. I'm fine with either distro as long as it passes the patch build.